e2e/cli_scan_test.go

```diff
@@ -68,16 +69,22 @@ func TestCLIScan(t *testing.T) {
 			// analyze image with --merge-sbom directory sbom, and export to backend
 			t.Logf("analyze image...")
 			analyzeImage(t, DirectoryAnalyzeOutputSBOMFile, appID)
+			// wait for refreshing materialized views
+			time.Sleep(common.WaitForMaterializedViewRefreshSecond * time.Second)
```

Comment:
I think it is better to check the db every 1 second in order to see if the the refresh has ended, I dont like entering a sleep to the tests

Reply:
How can we decide if the refresh is not ended or the refresh is ended but the results are not the expected?

Reply:
Is there is no indication of refresh ended?

Reply:
From the cli we cannot check the end of view refresh.